components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -540,17 +604,11 @@ func getConfigForArgsPostgresqlOrOpenSearch(args []string, remoteService string)
 		}
 		return dest, nil
 	}
-
-	return nil, nil
-
 }
 
 // isConfigChanged checks if configuration is changed
 func isConfigChanged(src interface{}, dest interface{}) bool {
-	if reflect.DeepEqual(src, dest) {
-		return false
-	}
-	return true
+	return !reflect.DeepEqual(src, dest)
```

Comment:
This behaviour is tested completely @ArvinthC3000 ?

Reply:
Yes